drivers/scheduler/openshift/openshift.go

```diff
@@ -57,6 +59,70 @@ func (k *openshift) StartSchedOnNode(n node.Node) error {
 	return nil
 }
 
+func (k *openshift) Schedule(instanceID string, options scheduler.ScheduleOptions) ([]*scheduler.Context, error) {
+	var apps []*spec.AppSpec
+	if len(options.AppKeys) > 0 {
+		for _, key := range options.AppKeys {
+			spec, err := k.SpecFactory.Get(key)
+			if err != nil {
+				return nil, err
+			}
+			apps = append(apps, spec)
+		}
+	} else {
+		apps = k.SpecFactory.GetAll()
+	}
+
+	var contexts []*scheduler.Context
+	for _, app := range apps {
+
+		appNamespace := app.GetID(instanceID)
+		k8sOps := k8s_ops.Instance()
+
+		// Update security context for namespace and user
+		if err := k.updateSecurityContextConstraints(k8sOps, appNamespace); err != nil {
+			return nil, err
+		}
+
+		specObjects, err := k.CreateSpecObjects(app, appNamespace, options.StorageProvisioner)
+		if err != nil {
+			return nil, err
+		}
+
+		ctx := &scheduler.Context{
+			UID: instanceID,
+			App: &spec.AppSpec{
+				Key:      app.Key,
+				SpecList: specObjects,
+				Enabled:  app.Enabled,
+			},
+		}
+
+		contexts = append(contexts, ctx)
+	}
+
+	return contexts, nil
+}
+
+func (k *openshift) updateSecurityContextConstraints(k8sOps k8s_ops.Ops, namespace string) error {
+	// Get priviledged context
+	context, err := k8sOps.GetSecurityContextConstraints("privileged")
+	if err != nil {
+		return err
+	}
+
+	// Add user and namespace to context
+	context.Users = append(context.Users, "system:serviceaccount:"+namespace+":default")
+
+	// Update context
+	_, err = k8sOps.UpdateSecurityContextConstraints(context)
+	if err != nil {
```

Comment:
nit: We can simplify this by just 
`return err`
If it's nil then we will return nil else error.

Reply:
I'm going to leave this be